### BIG differences, this is diff of wrong files.
### CLUS "Symbol LAMBDA"
### and CLHS topic selection http://clhs.lisp.se/Body/a_lambda.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:symbols:lambda.txt	2017-04-27 21:57:39.000000000 +0100
+++ saved-pages-clus/cl:symbols:lambda.txt	2017-04-27 21:57:39.000000000 +0100
@@ -1,4 +1,58 @@
-LAMBDA
-Please select which reference to LAMBDA you intended
+Symbol LAMBDA

-MacroSymbol
\ No newline at end of file
+Syntax
+ lambda lambda-list [[declaration* | documentation]] form*
+Arguments
+ lambda-list - an ordinary lambda list.
+ declaration - a declare expression; not evaluated.
+ documentation - a string; not evauated.
+ form - a form.
+Description
+A lambda expression is a list that can be used in place of a function name in certain contexts to denote a function by directly describing its behavior rather than indirectly by referring to the name of an establishestablished function.
+documentation is attached to the denoted function (if any is actually created) as a documentation string.
+See Also
+Macro LAMBDA, Special Operator FUNCTION, Generic Function DOCUMENTATION, {\secref\LambdaExpressions}, {\secref\LambdaForms}, {\secref\DocVsDecls}
+Notes
+The lambda form
+((lambda lambda-list . body) . arguments)
+is semantically equivalent to the function form
+(funcall #'(lambda lambda-list . body) . arguments)
+\
\ No newline at end of file
